notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>I think it would be nice to have some of the results from Devops of request time. We can also add that image to the OKR document</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>I think the scenario doc is the better place for that, but it's probably worth adding a cell with the results here at the very end. Will try to get to that later today.</p><p><br></p>